### PR TITLE
[App Service] Add support for Premium V3 pricing tier

### PR DESCRIPTION
**Pull request by vinisoto**
**Description**  

Adding support for PremiumV3 pricing tier

**Testing Guide**  

`appservice plan create --location westus --resource-group my-resource-group --name -my-new-app-service-plan --sku P1v3`

P1V3 => Premium V3 Small
P2V3 => Premium V3 Medium
P3V3 => Premium V3 Large

**History Notes**  

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by vinisoto** [comment]
cc: @panchagnula, @jvano

File: src/azure-cli/azure/cli/command_modules/appservice/utils.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def get_sku_name(tier):  # pylint: disable=too-many-return-statements
         return 'PREMIUM'
     if tier in ['P1V2', 'P2V2', 'P3V2']:
         return 'PREMIUMV2'
+    if tier in ['P1V3', 'P2V3', 'P3V3']:
+        return 'PREMIUMV3'
     if tier in ['PC2', 'PC3', 'PC4']:
         return 'PremiumContainer'
```
Comment (jvano) [inline src/azure-cli/azure/cli/command_modules/appservice/utils.py:46]: we won't GA PC SKU, so we should remove it.

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 2]
```diff
@@ -45,9 +45,9 @@ def load_arguments(self, _):
     # PARAMETER REGISTRATION
     name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
     sku_arg_type = CLIArgumentType(
-        help='The pricing tiers, e.g., F1(Free), D1(Shared), B1(Basic Small), B2(Basic Medium), B3(Basic Large), S1(Standard Small), P1V2(Premium V2 Small), PC2 (Premium Container Small), PC3 (Premium Container Medium), PC4 (Premium Container Large), I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
+        help='The pricing tiers, e.g., F1(Free), D1(Shared), B1(Basic Small), B2(Basic Medium), B3(Basic Large), S1(Standard Small), P1V2(Premium V2 Small), P1V3(Premium V3 Small), P2V3(Premium V3 Medium), P3V3(Premium V3 Large), PC2 (Premium Container Small), PC3 (Premium Container Medium), PC4 (Premium Container Large), I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
         arg_type=get_enum_type(
-            ['F1', 'FREE', 'D1', 'SHARED', 'B1', 'B2', 'B3', 'S1', 'S2', 'S3', 'P1V2', 'P2V2', 'P3V2', 'PC2', 'PC3',
+            ['F1', 'FREE', 'D1', 'SHARED', 'B1', 'B2', 'B3', 'S1', 'S2', 'S3', 'P1V2', 'P2V2', 'P3V2', 'P1V3', 'P2V3', 'P3V3', 'PC2', 'PC3',
              'PC4', 'I1', 'I2', 'I3']))
```
Comment (jvano) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:51]: same here

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 3]
```diff
@@ -45,9 +45,9 @@ def load_arguments(self, _):
     # PARAMETER REGISTRATION
     name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
     sku_arg_type = CLIArgumentType(
-        help='The pricing tiers, e.g., F1(Free), D1(Shared), B1(Basic Small), B2(Basic Medium), B3(Basic Large), S1(Standard Small), P1V2(Premium V2 Small), PC2 (Premium Container Small), PC3 (Premium Container Medium), PC4 (Premium Container Large), I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
+        help='The pricing tiers, e.g., F1(Free), D1(Shared), B1(Basic Small), B2(Basic Medium), B3(Basic Large), S1(Standard Small), P1V2(Premium V2 Small), P1V3(Premium V3 Small), P2V3(Premium V3 Medium), P3V3(Premium V3 Large), PC2 (Premium Container Small), PC3 (Premium Container Medium), PC4 (Premium Container Large), I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
```
Comment (jvano) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:48]: >PC2 (Premium Container Small), PC3 (Premium Container Medium), PC4 (Premium Container Large) [](start = 230, length = 92)

same here

**Review by jvano — APPROVED** [review]
:shipit:

File: src/azure-cli/azure/cli/command_modules/appservice/utils.py (hunk 1 above)
Comment (vinisoto) [reply src/azure-cli/azure/cli/command_modules/appservice/utils.py:46]: Yes. I will remove it in a following PR

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py (hunk 3 above)
Comment (vinisoto) [reply src/azure-cli/azure/cli/command_modules/appservice/_params.py:48]: Yes. I will remove it in a following PR

**Comment by vinisoto** [comment]
Will remove PremiumContainer in a following PR

**Review by pagariyaalok — APPROVED** [review]
:shipit:

**Comment by yonzhan** [comment]
App Service

**Review by qwordy — APPROVED** [review]

**Pull request merged**